pkg/ebpf/events_processor.go

```diff
@@ -446,3 +446,29 @@ func CopyFileByPath(src, dst string) error {
 	}
 	return nil
 }
+
+// CopyFileByRelativePath copies a file from src to dst, where destination is relative to a given directory
+func CopyFileByRelativePath(srcName string, dstDir *os.File, dstName string) error {
```

Comment:
Why not to place in utils together with other functions you created ? Something different here ?